source/CommandLine/main.cpp

```diff
@@ -46,36 +46,38 @@ int VIREO_MAIN(int argc, const char * argv[])
             if (strcmp(argv[arg], "-d") == 0)
                 gShells._pRootShell->DumpPrimitiveDictionary();
 			gShells._pUserShell = TypeManager::New(gShells._pRootShell);
```

Comment:
Consider adding curly braces for the previous if, it is not clear what statements belong to it.